ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs

```diff
@@ -149,137 +203,194 @@ void StartImportActivity(int resultConstant)
-            }
-            else if (button.Data.Code == ListItemCode.CheckRecognizer)
+        private void ScanWithSelectionOverlayBarcode()
+        {
+            var configuration = new BarcodeScannerConfiguration();
+            configuration.SetTopBarButtonsColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            configuration.SetSelectionOverlayConfiguration(new SelectionOverlayConfiguration(
+                overlayEnabled: true,
+                automaticSelectionEnabled: false,
+                textFormat: IO.Scanbot.Sdk.Barcode.UI.BarcodeOverlayTextFormat.CodeAndType
+            ));
+
+            var intent = BarcodeScannerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, QR_BARCODE_DEFAULT_REQUEST);
+        }
+
+        private void ScanMultipleBarcodes()
+        {
+            var configuration = new BatchBarcodeScannerConfiguration();
+            configuration.SetCameraZoomRatio(1f);
+            configuration.SetTopBarButtonsColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            configuration.SetDetailsBackgroundColor(Color.White);
+            configuration.SetDetailsActionColor(Color.White);
+            configuration.SetDetailsBackgroundColor(Color.Gray);
+            configuration.SetDetailsPrimaryColor(Color.White);
+            configuration.SetBarcodesCountTextColor(Color.White);
+
+            configuration.SetOrientationLockMode(CameraOrientationMode.Portrait);
+
+            configuration.SetFinderTextHint("Please align the QR-/Barcode in the frame above to scan it.");
+            var intent = BatchBarcodeScannerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, MULTIPLE_QR_BARCODES_REQUEST);
+        }
+
+        private void ScanMrz()
+        {
+            var configuration = new MRZScannerConfiguration();
+            configuration.SetSuccessBeepEnabled(false);
+            configuration.SetTopBarButtonsColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            var intent = MRZScannerActivity.NewIntent(this, configuration);
+            StartActivityForResult(intent, SCAN_MRZ_REQUEST);
+        }
+
+        private void ScanGenericDocument()
+        {
+            var configuration = new GenericDocumentRecognizerConfiguration();
+            configuration.SetAcceptedDocumentTypes(new List<RootDocumentType>
             {
-                var config = new CheckRecognizerConfiguration();
-                config.SetAcceptedCheckStandards(new List<IO.Scanbot.Check.Entity.RootDocumentType>
+                RootDocumentType.DeIdCardFront,
+                RootDocumentType.DeIdCardBack,
+            });
+
+            configuration.SetTopBarButtonsInactiveColor(Color.White);
+            configuration.SetTopBarBackgroundColor(Color.Black);
+
+            //configuration.SetFieldsDisplayConfiguration(
```

Comment:
@yurii-scanbot, is this comment for illustration purposes, or is the code redundant? If the code is useful for the example, perhaps you can add an additional comment explaining what it is for (for helping our customers) otherwise you may prefer to remove it.